ci/pod/docker-compose.plugin.yml

```diff
@@ -104,17 +104,22 @@ services:
   kafka-server2:
     image: bitnami/kafka:2.8.1
     env_file:
-      - ci/pod/kafka/kafka-server/env/common.env
+      - ci/pod/kafka/kafka-server/env/common2.env
     environment:
       KAFKA_CFG_ZOOKEEPER_CONNECT: zookeeper-server2:2181
     restart: unless-stopped
     ports:
       - "19092:9092"
+      - "19094:9094"
     depends_on:
       - zookeeper-server1
       - zookeeper-server2
     networks:
       kafka_net:
+    volumes:
+      - ./ci/pod/kafka/kafka-server/kafka_jaas.conf:/opt/bitnami/kafka/config/kafka_jaas.conf:ro
+      - ./ci/pod/kafka/kafka-server/selfsigned.jks:/opt/bitnami/kafka/config/certs/kafka.keystore.jks:ro
+      - ./ci/pod/kafka/kafka-server/selfsigned.jks:/opt/bitnami/kafka/config/certs/kafka.truststore.jks:ro
```

Comment:
The `selfsigned.jks` file is only generated when the `last` test is executed, look: https://github.com/apache/apisix/blob/master/.github/workflows/build.yml#L117-L120, and should not exist when the `plugin` test is executed. Why can CI still pass, very puzzled.....

Reply:
Sorry.My fault.

> Why can CI still pass, very puzzled....

The file or directory does not need to exist on the Docker host already. It is created on demand if it does not yet exist.
So it wouldn't influence the CI.

refer: https://docs.docker.com/storage/bind-mounts/

Reply:
Got it ~~~